### PR TITLE
add subtract operation

### PR DESCRIPTION
**Pull request by drntam**

### CODE REVIEW

File: src/calculator.ts [hunk 1]
```diff
@@ -2,4 +2,8 @@ export class Calculator {
   public add(x: number, y: number) : number {
     return x + y;
   }
+
+  public subtract(x: number, y: number) : number {
+    return x - y;
```
Comment (markdstouffer) [inline src/calculator.ts:7]: genius work

File: src/index.ts [hunk 2]
```diff
@@ -5,3 +5,5 @@ const calc = new Calculator();
 const result_add = calc.add(2, 3);
 console.log(`result_add = ${result_add}`);
 
+const result_subtract = calc.subtract(5,3);
+console.log(`result_subtract = ${result_subtract}`);
```
Comment (markdstouffer) [inline src/index.ts:9]: Looks good to me.

**Review by markdstouffer — APPROVED** [review]

File: src/calculator.ts (hunk 1 above)
Comment (MichaelGadhia) [reply src/calculator.ts:7]: Looks perfect

File: src/index.ts (hunk 2 above)
Comment (MichaelGadhia) [reply src/index.ts:9]: Seems correct

**Pull request merged**